recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.5.1" %}
```

Comment:
Could you build the latest v0.5.2 https://pypi.org/project/whylabs-client/0.5.2/ ?